### DISCUSSION TITLE
중복되는 interface 빼기

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#49] [05. General]
<img width="314" alt="스크린샷 2022-08-01 오후 2 31 56" src="https://user-images.githubusercontent.com/72402747/182078653-5047493f-ba44-40a2-a6cf-66127ce30616.png">

<img width="314" alt="스크린샷 2022-08-01 오후 2 32 03" src="https://user-images.githubusercontent.com/72402747/182078678-fd476600-46b3-4a23-a661-97ffc09e6d68.png">

만약 겹치는 interface가 있다면 따로 빼두는게 좋을 거 같습니당!!

**Comment by punchdrunkard** [comment]
# 담당
혜경님(@Hyevvy)

# 관련 PR 및 Issue
- #54
- #56
